audiofile/core/convert.py

```diff
@@ -36,7 +36,8 @@ def convert_to_wav(
         offset: start reading at offset in seconds
 
     Raises:
-        RuntimeError: if ``file`` is broken or not a supported format
+        RuntimeError: if ``file`` is missing,
+            broken or not a supported format
```

Comment:
```suggestion
            broken or format is not supported
```

I think this sounds more correct than "if file is not a supported format"

Reply:
I updated all docstrings accordingly and also the screenshot in the description of this pull request.